File: src/main/java/br/com/leonardoferreira/jirareport/domain/vo/IssuePeriodDetails.java
package br.com.leonardoferreira.jirareport.domain.vo;

import lombok.Builder;
import lombok.Data;

@Data
@Builder
public class IssuePeriodDetails {

    private Double avgLeadTime;

    private Integer issueCount;

    private Long boardId;

    private String jql;

    private Double wipAvg;

    private Double avgPctEfficiency;

}

File: src/main/java/br/com/jiratorio/domain/IssuePeriodDetails.kt
package br.com.jiratorio.domain

data class IssuePeriodDetails(
    var avgLeadTime: Double? = null,
    var issueCount: Int? = null,
    var boardId: Long? = null,
    var jql: String? = null,
    var wipAvg: Double? = null,
    var avgPctEfficiency: Double? = null
)
